### DISCUSSION TITLE
Using re-frame-10x with re-posh

### DISCUSSION THREAD

**Discussion by filipesilva** [Day8/re-frame-10x#291] [Q&A] [answered]
Heya,

https://github.com/denistakeda/re-posh adds datascript subscription support to re-frame. It uses a datascript database for these subscriptions instead of re-frame's `app-db`.

Is there a good way to see this database in re-frame-10x?

Thanks!

**Comment by mike-thompson-day8** [comment, marked as answer]
`app-db` is currently a map (inside a `reagent/atom`).  Whereas a `datascript` database is an **_entirely_** different beast.  

So, I guess the simple answer is "no, can't be done".

**Comment by filipesilva** [comment]
Gotcha, thanks for the quick reply!
